docs/aws-sdk-for-java-with-minio.md

```diff
@@ -6,113 +6,145 @@
 
 Install Minio Server from [here](http://docs.minio.io/docs/minio).
 
-## 2. Installation
-
-Install `aws-sdk-java` from AWS SDK for Java official docs [here](https://aws.amazon.com/sdk-for-java/), for this document we are using the aws-sdk-java [ZIP](https://sdk-for-java.amazonwebservices.com/latest/aws-java-sdk.zip)
+## 2. Setup dependencies
+
+You can either download and install the [aws-java-sdk](https://sdk-for-java.amazonwebservices.com/latest/aws-java-sdk.zip) / [AWS Java SDK documentation](https://aws.amazon.com/sdk-for-java/) or use [Apache Maven](https://maven.apache.org/) to get the needed dependencies.
+
+```xml
+…
```

Comment:
are these ... intended here?

Reply:
Yes because the listed xml is just an excerpt. I usually add those dots to indicate that.

Reply:
I see..that makes sense.